### PR TITLE
landing page of ecommerce project

### PR DESCRIPTION
**Pull request by devAmlan**
This is the homepage of my ecommerce project. Please review my PR.

### CODE REVIEW

**Comment by Ranjeetk489** [comment]
the preview link seems to be broken

File: css/header.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
+.header{
+    display:flex;
+    justify-content:space-between;
+    align-items:center;
+    height:55px;
```
Comment (Ranjeetk489) [inline css/header.css]: use rem

File: css/header.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      width:90px;
+      height:33px;
+      background-color:#8991a5;
+      color:#fff;
+      border:none;
+      border-radius:8px;
+      font-size:1.6rem;
+      margin:1rem;
+      text-align:center;
+      font-weight:700;
```
Comment (Ranjeetk489) [inline css/header.css]: use var

File: css/header.css [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      position: relative;
+  }
+  .badge{
+      position:absolute;
+      top:-6px;
+      left:15px;
+      font-size:15px;
+      width:18px;
+      height:18px;
+      background-color:firebrick;
```
Comment (Ranjeetk489) [inline css/header.css]: how is it working here??

**Review by Ranjeetk489 — APPROVED** [review]
Use var, rem and remove empty spaces after applying these it'll be good to go

File: pages/home/home.css [hunk 4]
```diff
@@ -0,0 +1,342 @@
...
+   justify-content:center;
+   align-items:center;
+}
+.header_nav ul li{
+   margin:10px;
+   list-style:none;
+   font-size:2.5rem;
+}
+.login_nav{
+    width:90px;
```
Comment (developer-anand) [inline pages/home/home.css]: you can use rem instead of px.

**Review by developer-anand — CHANGES_REQUESTED** [review]
Just be consistent try to use rem when less than 10px.

File: pages/home/home.css (hunk 4 above)
Comment (devAmlan) [reply pages/home/home.css]: I will change to rem , thanks anand.

File: css/header.css (hunk 2 above)
Comment (devAmlan) [reply css/header.css]: yeah i will change to var  , thanks ranjeet

File: css/header.css (hunk 3 above)
Comment (devAmlan) [reply css/header.css]: I don't understand your point. Can u plz elaborate more on what I have done wrong?

File: css/header.css (hunk 1 above)
Comment (devAmlan) [reply css/header.css]: yeah i will change it , thanks ranjeet

**Pull request merged**